resources/leiningen/new/kraken_http_api/Dockerfile

```diff
@@ -1,13 +1,20 @@
-FROM quay.io/democracyworks/didor:latest
+FROM quay.io/democracyworks/clojure-yourkit:lein-2.7.1
 
 RUN mkdir -p /usr/src/{{name}}
 WORKDIR /usr/src/{{name}}
 
 COPY project.clj /usr/src/{{name}}/
 
-RUN lein deps
+ARG env=production
+
+RUN lein with-profile $env deps
 
 COPY . /usr/src/{{name}}
 
-RUN lein test
-RUN lein immutant war --name {{name}} --destination target --nrepl-port=11111 --nrepl-start --nrepl-host=0.0.0.0
+RUN lein with-profiles $env,test test
+RUN lein with-profile $env uberjar
+
+CMD java ${JVM_OPTS:--XX:+UseG1GC} \
+    -javaagent:resources/jars/com.newrelic.agent.java/newrelic-agent.jar \
```

Comment:
Maybe we should get the cgroup heap stuff in here too? And in the works template?